lib/filter_cli.c

```diff
@@ -174,16 +177,21 @@ DEFPY_YANG(
 	 */
 	snprintf(xpath, sizeof(xpath),
 		 "/frr-filter:lib/access-list[type='ipv4'][name='%s']", name);
-	nb_cli_enqueue_change(vty, xpath, NB_OP_CREATE, NULL);
 	if (seq_str == NULL) {
 		/* Use XPath to find the next sequence number. */
 		sseq = acl_get_seq(vty, xpath);
+		if (sseq < 0) {
+			vty_out(vty, "%% Malformed sequence value\n");
```

Comment:
We can print this in `acl_get_seq()` because we have VTY, right?

Reply:
I thought about it.
If so, it (`acl_get_seq()`)  will be not pure. Need to add parameters so that it does not print these error messages when deleting. As said in commit log, no change in deleting.
```
static int64_t acl_get_seq(struct vty *vty, const char *xpath, bool show)
```
But printing  in `acl_get_seq()` can compact code. Anyway is okay to me. Needed? :)

Reply:
But it's used only where `vty` is never NULL, right?

Reply:
Updated, please help review it, thanks.